editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -636,13 +636,15 @@ void VisualShaderEditor::_duplicate_nodes() {
 
 	for (int i = 0; i < graph->get_child_count(); i++) {
 
-		if (Object::cast_to<GraphNode>(graph->get_child(i))) {
+		GraphNode *gn = Object::cast_to<GraphNode>(graph->get_child(i));
+		if (gn) {
 			int id = String(graph->get_child(i)->get_name()).to_int();
```

Comment:
Maybe replace `graph->get_child(i)` by `gn` here?

Reply:
This is necessary because `GraphEdit` has some other control nodes like `HScrollBar` as direct children, this will lead to crash.

Reply:
Ah, yes, makes sense, I looked at the wrong line.

Reply:
There are other places when the pointer is not reused here, but that can be fixed as part of other PR pertaining to the issue I linked...